contracts/libraries/range/pool/MintRangeCall.sol

```diff
@@ -42,8 +42,10 @@ library MintRangeCall {
         // check for invalid receiver
         if (params.to == address(0))
             require(false, "CollectToZeroAddress()");
-        
-        // initialize cache
+
+        // validate position ticks
+        ConstantProduct.checkTicks(params.lower, params.upper, cache.constants.tickSpacing);
+
```

Comment:
`ConstantProduct.checkTicks()` is now called at the start of `MintRangeCall.sol`.